runtime/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaProxyFactory.java

```diff
@@ -75,7 +75,7 @@ public void attach(
         MqttKafkaBindingConfig kafkaBinding = new MqttKafkaBindingConfig(binding);
         bindings.put(binding.id, kafkaBinding);
 
-        factories.values().forEach(streamFactory -> streamFactory.onAttached(binding.id));
+        factories.values().forEach(streamFactory -> streamFactory.onAttached(binding.id, binding.name));
```

Comment:
Is this change necessary?

Expecting that `binding.id` is sufficient to extract both namespace and local name of binding via context, agree?